docs/docs/react.md

```diff
@@ -0,0 +1,19 @@
+---
+title: React
+---
+
+Learn about React, including what it is, why you might use it, and how it fits into the Gatsby ecosystem.
+
+## What is React?
+
+React (sometimes written as React.js, or ReactJS) is a code library for building web sites and applications. It's written using [JavaScript](https://www.gatsbyjs.org/docs/glossary#javascript), one of the programming languages used to create web pages.
+
+Facebook first released React in 2013. The company maintains the project, along with a community of contributors. React, like Gatsby, is free to use and open source under the terms of the [MIT License](https://github.com/facebook/react/blob/master/LICENSE).
+
+React lets you create _components_ &mdash; reusable, shareable combinations of JavaScript, CSS, and HTML or SVG. You might, for example, build an `InputBox` component that includes an input field, label, and validation. You can then use the `InputBox` component on multiple pages, and across multiple projects. In other words, Components help you build web sites and single-page applications more quickly.
+
+React is one of the foundational building blocks of Gatsby. Gatsby bundles React, [webpack](https://www.gatsbyjs.org/docs/glossary#webpack), [GraphQL](https://www.gatsbyjs.org/docs/glossary#graphql), as well as other libraries and tools into a single framework for building web sites. Rather than installing and configuring a development environment from scratch, you can install Gatsby and start building.
```

Comment:
```suggestion
React is one of the foundational building blocks of Gatsby. Gatsby bundles React, [webpack](/docs/glossary#webpack), [GraphQL](/docs/glossary#graphql), as well as other libraries and tools into a single framework for building web sites. Rather than installing and configuring a development environment from scratch, you can install Gatsby and start building.
```